CHANGELOG.md

```diff
@@ -1,3 +1,121 @@
+- Node-local-dns is now using UDP for external queries [#6796](https://github.com/kubermatic/kubermatic/issues/6796) 
+- Add validation for Kubermatic Constraint Template API. [#6841](https://github.com/kubermatic/kubermatic/issues/6841) 
+- Fetch the provisioning cloud-init over the api-server  [#6843](https://github.com/kubermatic/kubermatic/issues/6843) 
+- Add `FELIX_IGNORELOOSERPF=true` to `calico-node` container env to allow running on nodes with `net.ipv4.conf.*.rp_filter = 2` set. [#6865](https://github.com/kubermatic/kubermatic/issues/6865) 
+- Hetzner AMD Cloud Server (CPX) now selectable when creating a user cluster [#6872](https://github.com/kubermatic/kubermatic/issues/6872)
+- Add GPU support for Azure provider [#6605](https://github.com/kubermatic/kubermatic/issues/6605) ([zreigz](https://github.com/zreigz))
+
+### Bugfixes
+
+- Fix kube-system/coredns PodDisruptionBudget matchLabels in user clusters [#6398](https://github.com/kubermatic/kubermatic/issues/6398) 
+- Fix S3 storage uploader CA bundle option flag [#6732](https://github.com/kubermatic/kubermatic/issues/6732) 
+- Fix cases where GET and LIST endpoints for Kubermatic Constraints failed or didn't return all results because there were no related synced Gatekeeper Constraints on the user cluster by just taking the Status from the Gatekeeper Constraints and setting the Synced status to false if the Gatekeeper Constraint is missing. [#6800](https://github.com/kubermatic/kubermatic/issues/6800) 
+- Fix KAS service port in Tunneling agent configuration. [#6569](https://github.com/kubermatic/kubermatic/issues/6569) 
+- Fix a bug in OPA-integration where deleting a Constraint Template in the seed cluster, when the user cluster Constraint Template is already deleted caused the deletion to get stuck.Fixed a bug in OPA-integration where creating a cluster with OPA-integration enabled didn't trigger the Constraint Template reconcile loop. [#6580](https://github.com/kubermatic/kubermatic/issues/6580) 
+- Fix issue with gatekeeper not recognizing the AdmissionReview v1 version by changing the webhook to use v1beta1 [#6550](https://github.com/kubermatic/kubermatic/issues/6550) 
+- Fix a bug with kubermatic constraints delete getting stuck when corresponding user cluster constraint is missing [#6598](https://github.com/kubermatic/kubermatic/issues/6598) 
+- Fix CE installer binary in EE downloads [#6673](https://github.com/kubermatic/kubermatic/issues/6673) 
+- Fix nodeport-proxy role used with LoadBalancer expose strategy. [#6646](https://github.com/kubermatic/kubermatic/issues/6646) 
+- Fix the operator failing to reconcile the ValidatingWebhookConfiguration object for the cluster validation webhook [#6639](https://github.com/kubermatic/kubermatic/issues/6639) 
+- Fix installer trying an invalid certificate to test cert-manager [#6761](https://github.com/kubermatic/kubermatic/issues/6761) 
+
+### Updates
```

Comment:
I don't see the cert-manager update to 1.2.0 here (#6739). Is that intentional? The PR also has `action required` in the release note block.